README.md

```diff
@@ -1,17 +1,19 @@
-# Stacks Wallet
+# Stacks Wallet for Desktop
 
 ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/blockstack/stacks-wallet/Build)
 
 ![Stacks Wallet Hero](/resources/readme.png)
 
-Implementation of the Stacks 2.0 wallet
+Implementation of the Stacks 2.0 wallet for Desktop
 
 ## Getting started
 
-```sh
-yarn
-```
+1. Ensure you have [Yarn](https://yarnpkg.com/) installed.
+1. After [cloning the repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository), run `yarn` in the root of the project to install the dependencies.
+1. Run `yarn dev:testnet` to start a development server.
```

Comment:
```suggestion
3. Run `yarn dev:testnet` to start a development server.
```

Reply:
I don't think we need to number them, Markdown takes `1.` as an `ol`